lib/block_view.go

```diff
@@ -899,6 +1015,15 @@ type UtxoOperation struct {
 	// For disconnecting diamonds.
 	PrevDiamondEntry *DiamondEntry
 
+	// For disconnecting NFTs.
+	PrevNFTEntry *NFTEntry
+	// We need the new NFT Entry to remove ownership mappings
+	NewNFTEntry               *NFTEntry
+	PrevNFTBidEntry           *NFTBidEntry
+	DeletedNFTBidEntries      []*NFTBidEntry
```

Comment:
Hmm... these slices look a little sus but will look at their usage and come back.